CRITICAL - 2025-10-29 07:00:24 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "Signup.php"
[Method: GET, Route: signup]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Signup.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('Signup', [], true)
 3 APPPATH\Controllers\Auth.php(55): view('Signup')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->signup()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\EXPAND\\public\\index.php')
ERROR - 2025-10-29 07:13:17 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\EXPAND\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\EXPAND\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\EXPAND\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\EXPAND\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\EXPAND\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\EXPAND\app\Models\UserModel.php(71): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\EXPAND\app\Controllers\Auth.php(41): App\Models\UserModel->verifyLogin('jerwinagustin20...', '094628')
#8 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(933): App\Controllers\Auth->loginProcess()
#9 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\EXPAND\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\EXPAND\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\EXPAND\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\EXPAND\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\EXPAND\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\EXPAND\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\EXPAND\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\EXPAND\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\EXPAND\app\Models\UserModel.php(71): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\EXPAND\app\Controllers\Auth.php(41): App\Models\UserModel->verifyLogin('jerwinagustin20...', '094628')
#7 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(933): App\Controllers\Auth->loginProcess()
#8 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\EXPAND\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\EXPAND\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\EXPAND\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\EXPAND\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-10-29 07:13:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\UserModel.php(71): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Auth.php(41): App\Models\UserModel->verifyLogin('jerwinagustin200@gmail.com', '094628')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->loginProcess()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\EXPAND\\public\\index.php')
CRITICAL - 2025-10-29 07:19:42 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "Settings.php"
[Method: GET, Route: settings]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Settings.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('Settings', [], true)
 3 APPPATH\Controllers\Pages.php(24): view('Settings')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pages->settings()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\EXPAND\\public\\index.php')
ERROR - 2025-10-29 07:27:55 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\EXPAND\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\EXPAND\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\EXPAND\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\EXPAND\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\EXPAND\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('jerwinagustin03...', 'users.email', Array)
#6 C:\xampp\htdocs\EXPAND\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('jerwinagustin03...', 'users.email', Array, NULL, 'email')
#7 C:\xampp\htdocs\EXPAND\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'jerwinagustin03...', Array, Array, 'email')
#8 C:\xampp\htdocs\EXPAND\app\Controllers\Auth.php(76): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(933): App\Controllers\Auth->signupProcess()
#10 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#11 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\EXPAND\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\EXPAND\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\EXPAND\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\EXPAND\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\EXPAND\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\EXPAND\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\EXPAND\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\EXPAND\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\EXPAND\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('jerwinagustin03...', 'users.email', Array)
#5 C:\xampp\htdocs\EXPAND\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('jerwinagustin03...', 'users.email', Array, NULL, 'email')
#6 C:\xampp\htdocs\EXPAND\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'jerwinagustin03...', Array, Array, 'email')
#7 C:\xampp\htdocs\EXPAND\app\Controllers\Auth.php(76): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(933): App\Controllers\Auth->signupProcess()
#9 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\EXPAND\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\EXPAND\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\EXPAND\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\EXPAND\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\EXPAND\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-10-29 07:27:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: signup]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('jerwinagustin032@gmail.com', 'users.email', [...])
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('jerwinagustin032@gmail.com', 'users.email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'jerwinagustin032@gmail.com', [...], [...], 'email')
 7 APPPATH\Controllers\Auth.php(76): CodeIgniter\Validation\Validation->run()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->signupProcess()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\EXPAND\\public\\index.php')
